src-cpp/TopicImpl.cpp

```diff
@@ -33,15 +33,15 @@
 
 #include "rdkafkacpp_int.h"
 
-const int32_t RdKafka::Topic::PARTITION_UA;
+const int32_t RdKafka::Topic::PARTITION_UA = -1;
```

Comment:
With these moved from the cpp header file can we use the C defines from rdkafka.h for these?